tiny_dnn/core/kernels/conv2d_grad_op_avx.h

```diff
@@ -19,1092 +19,697 @@ namespace kernels {
 
 // float ver
 template <typename Allocator>
-inline void accumulate_db(
-    const index3d<serial_size_t>&        out,
-    const std::vector<float, Allocator>& curr_delta,
-    std::vector<float, Allocator>&       db
-) {
-    if (out.width_ == 1 && out.height_ == 1) {
-        size_t nblocks = out.depth_ / 8;
-        for (size_t i = 0; i < nblocks; ++i) {
-            _mm256_storeu_ps(
-                &db[i * 8],
-                _mm256_add_ps(
-                    _mm256_loadu_ps(&db[i * 8]),
-                    _mm256_loadu_ps(&curr_delta[i * 8])
-                )
-            );
-        }
-        for (size_t outc = nblocks * 8;
-             outc < out.depth_;
-             ++outc
-        ) {
-            db[outc] += curr_delta[outc];
-        }
-    } else {
-        auto area = out.area();
-        size_t nblocks = area / 8;
-        size_t remainder = area & 7;
-        // prepare load-mask beforehand
-        static const int32_t masks[] = {
-            -1, -1, -1, -1,
-            -1, -1, -1, -1,
-            0, 0, 0, 0,
-            0, 0, 0, 0,
-        };
-        __m256i mask = _mm256_loadu_si256(
-            (const __m256i*)(masks + 8 - remainder)
-        );
-        for (size_t outc = 0; outc < out.depth_; ++outc) {
-            serial_size_t idx = out.get_index(
-                0,
-                0,
-                static_cast<serial_size_t>(outc)
-            );
-            const float* delta = &curr_delta[idx];
-            __m256 sum0 = _mm256_setzero_ps();
-            __m256 sum1 = _mm256_setzero_ps();
-            for (size_t i = 0; i < nblocks / 2; ++i) {
-                sum0 = _mm256_add_ps(
-                    sum0,
-                    _mm256_loadu_ps(delta + i * 16)
-                );
-                sum1 = _mm256_add_ps(
-                    sum1,
-                    _mm256_loadu_ps(delta + i * 16 + 8)
-                );
-            }
-            if (nblocks & 1) {
-                sum0 = _mm256_add_ps(
-                    sum0,
-                    _mm256_loadu_ps(delta + (nblocks - 1) * 8)
-                );
-            }
-            sum0 = _mm256_add_ps(sum0, sum1);
-            sum1 = _mm256_loadu_ps(delta + nblocks * 8);
-            sum1 = _mm256_and_ps(sum1, _mm256_castsi256_ps(mask));
-            sum0 = _mm256_add_ps(sum0, sum1);
-            db[outc] += _mm_cvtss_f32(hsum256_ps(sum0));
-        }
+inline void accumulate_db(const index3d<serial_size_t>& out, const std::vector<float, Allocator>& curr_delta,
+                          std::vector<float, Allocator>& db) {
```

Comment:
The first two arguments of `accumulate_db` are listed in a same line. That is not favourable to me. (though number of characters in a line is expanded to 120 from 80)